tests/integration/tier0/inhibit-if-kmods-is-not-supported/test_kmods_not_supported.py

```diff
@@ -61,16 +75,41 @@ def test_do_not_inhibit_if_module_is_not_loaded(shell, convert2rhel):
             c2r.sendline("y")
             prompt_amount -= 1
 
-        assert c2r.expect("Kernel modules are compatible.") == 0
+        # Stop conversion before the point of no return as we do not need to run the full conversion
+        # with the check for kompatible kernel modules being right before the PONR.
+        assert c2r.expect("All loaded kernel modules are available in RHEL.") == 0
         c2r.expect("Continue with the system conversion?")
         c2r.sendline("n")
         assert c2r.exitstatus != 0
 
 
-def test_tainted_kernel_inhibitor(shell, convert2rhel):
-    # This test marks the kernel as tainted which is not supported by convert2rhel.
+def test_do_not_inhibit_if_module_is_force_loaded(shell, convert2rhel):
+    """
+    Test force loads kmod and checks for Convert2RHEL being inhibited.
+    """
+    # Force load the kernel module
+    assert shell("modprobe -f -v bonding").returncode == 0
```

Comment:
@danmyway, If this is failing only on CentOS7/OL7 you can add a simple condition and skip this test case scenario on those el7 distros.

Unless you find a solution to make it work properly on EL7 distros.

Reply:
Yep, that workaround would be needed for now.